sniffr/dog_routes/dog_routes.py

```diff
@@ -26,37 +35,39 @@ def post_dog():
 
     # If dog_id not in body then they are trying to create
     # If dog_id in body then updating content
-    if content["dog_id"]:
-        # update dog
-        ...
+    if "dog_id" in content.keys():
+        queried_dog = db.session.query(Dog).filter_by(dog_id=content['dog_id']).all()
+        if queried_dog:
+            # update dog
+            payload = jsonify(
+                {"Message": f"Successfully pinged API but editing dog id #{content['dog_id']} is not available yet."}
+            )
+            response = make_response(jsonify({"Message": f"Successfully pinged API but editing dog id #{content['dog_id']} is not available yet."}), 200)
+            response.headers["Content-Type"] = "application/json"
+            return response
+        
+        else:
+            payload = jsonify({"Message": "Dog Not Found"})
+            response = make_response(payload, 400)
```

Comment:
Yay 400 errors! 💃 🥂 🥳

Reply:
I'm sure I can produce more errors than that! 😈

Reply:
🤣